### PR TITLE
[IMP] sales: updated pro-forma invoice doc for 16

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3470564&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### FWP up to `master`

### CODE REVIEW

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (jero-odoo) [inline content/applications/sales/sales/invoicing/proforma.rst:18]:
```suggestion
To enable this feature, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
```

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (jero-odoo) [inline content/applications/sales/sales/invoicing/proforma.rst:30]: After some testing it looks like you can't send a pro-forma invoice on a SO/quotation for if you have already sent an invoice for a down payment (or a recurring subscription, so the button will not appear in that case. May be worth a call out. (You can send one for services/event registrations/courses/new subscriptions, so it doesn't seem to be limited to physical/consumable/storable goods)

**Review by jero-odoo — APPROVED** [review]
@ksc-odoo great job! Just one real note, so I am pushing this through with approval 👍

**Comment by ksc-odoo** [comment]
Thanks for the feedback, @jero-odoo 👍 I made all your suggested edits, and since you've already approved this PR (w/ those changes), I'll pass it on to the Materials team for a 2nd peer review ... so, this one's ready for ya: @hojo-odoo @Felicious @brse-odoo

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/proforma.rst:10]:
```suggestion
during importation, for customs purposes. They differ from a normal invoice, in that they are *not* a
```

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/proforma.rst:39]:
```suggestion
   Pro-forma invoices can **not** be sent for a sales order or quotation if an invoice for a down payment
   has already been sent, or for a recurring subscription. In either case, the :guilabel:`Send Pro-Froma Invoice` 
   button will not appear.
```

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/proforma.rst:44]:
```suggestion
When the :guilabel:`Send Pro-Forma Invoice` button is clicked, a pop-up window appears, from which an email can be sent.
```

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (hojo-odoo) [inline content/applications/sales/sales/invoicing/proforma.rst:48]:
```suggestion
sales order or quotation. The :guilabel:`Subject` field and the body of the
email can be modified, if necessary.
```

**Review by hojo-odoo — APPROVED** [review]
Hey @ksc-odoo,

Good job on this. Just a few comments for you to take a look at before tagging for final review.

Thanks,
hojo

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/invoicing/proforma.rst:10]: I don't believe a comma is necessary in this instance, as its a fairly short, straightforward sentence. So, I think I'll leave it as-is. However, if you believe I am incorrect, just let me know, and I'll make the change right away. Thanks!

**Comment by ksc-odoo** [comment]
Thanks for the helpful feedback @hojo-odoo -- I made those suggested edits, and believe this one is now ready for final review from @odoo/us-doc-review 👍

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/proforma.rst:71]: No need to use a bullet point for a single seealso item, and please update the path to be relative 🙏
```suggestion
   :doc:`invoicing_policy`
```

File: content/applications/sales/sales/invoicing/proforma/pro-forma-email-message-popup.png
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/proforma/pro-forma-email-message-popup.png:1]: Although this image is the right size, it could be retaken to remove a lot of the empty space on the right by resizing the browser window or by cropping this image.

File: content/applications/sales/sales/invoicing/proforma/pro-forma-setting.png
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/proforma/pro-forma-setting.png:1]: I don't think it is necessary to show the entire UI in the screenshot for this setting, please retake this image to remove the unnecessary information 🙂

File: content/applications/sales/sales/invoicing/proforma/send-pro-forma-invoice-button.png
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/proforma/send-pro-forma-invoice-button.png:1]: This image can be retaken with less width by resizing the browser window, this will help focus on the smartbutton and reduce the unnecessary parts of the UI being shown

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo, nice job with this PR! I have a few technical corrections and suggestions. Please see below and then tag me for another look once ready! Thank you 🙂

**Comment by ksc-odoo** [comment]
Thanks for your always-helpful feedback @samueljlieber -- I made all your suggested adjustments, so this one is ready for another quick review from you whenever you get a chance. Once I get your approval, I'll pass it along to the Final Review stage. Thanks again 👍

File: content/applications/sales/sales/invoicing/proforma.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/proforma.rst:19]: The ampersand does not appear to break the rst, but let's escape it just in case
```suggestion
in the :guilabel:`Quotations \& Orders` section, click the checkbox next to :guilabel:`Pro-Forma
```

File: content/applications/sales/sales/invoicing/proforma/pro-forma-email-message-popup.png
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/proforma/pro-forma-email-message-popup.png:1]: This image needs to be compressed :)

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo, thanks for implementing my previous suggestions! I caught two more small fixed, which I will push up now to move this PR forward for merge. Thank you!

**Review by samueljlieber — APPROVED** [review]
Hi @StraubCreative this PR is good to go 👍

**Review by StraubCreative — APPROVED** [review]

**Pull request closed without merging**
